docs/pages/includes/database-access/cloudsql_service_credentials.mdx

```diff
@@ -0,0 +1,19 @@
+The Teleport Database Service must have credentials of `teleport-db-service`
+GCP service account we created in step 3 in order to be able to generate IAM auth
+tokens.
+
+If the Teleport Database Service is hosted on a VM, you can [change the
+attached service
+account](https://cloud.google.com/compute/docs/instances/change-service-account).
+
+Alternatively, set the `GOOGLE_APPLICATION_CREDENTIALS` environment variable to
+point to the JSON credentials file you downloaded earlier. If you are using
+`systemd` to start `teleport`, then you should edit the service's
+`EnvironmentFile` to include the following env var:
+```code
+$ echo 'GOOGLE_APPLICATION_CREDENTIALS=/path/to/credentials.json' | sudo tee -a /etc/default/teleport
+```
+
+See [Authenticating as a service
+account](https://cloud.google.com/docs/authentication/production) in the Google
+Cloud documentation for more details.
```

Comment:
I would specify the additional details a user would expect to find on this page so they can get through this guide as quickly as possible. Otherwise, our style guide advises against linking to external guides.

Reply:
moved to next steps